client/src/components/CardsCarousel/CardCarousel.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import Carousel from "react-material-ui-carousel";
+import Card from "@material-ui/core/Card";
+import {
+	Button,
+	CardContent,
+	CardMedia,
+	IconButton,
+	Typography,
+	Grid,
+	Container,
+} from "@material-ui/core";
+import { CardActions } from "@mui/material";
+import ThumbUpOffAltIcon from "@mui/icons-material/ThumbUpOffAlt";
+import ThumbDownOffAltIcon from "@mui/icons-material/ThumbDownOffAlt";
+import ShareIcon from "@material-ui/icons/Share";
+
+
+const energisers = [
+	{
+		id: 1,
+		title: "Energiser 1",
+		description: "Energiser 1 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 2,
+		title: "Energiser 2",
+		description: "Energiser 2 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 3,
+		title: "Energiser 3",
+		description: "Energiser 3 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 4,
+		title: "Energiser 4",
+		description: "Energiser 4 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 5,
+		title: "Energiser 5",
+		description: "Energiser 5 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 6,
+		title: "Energiser 6",
+		description: "Energiser 6 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+	{
+		id: 7,
+		title: "Energiser 7",
+		description: "Energiser 7 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+];
```

Comment:
That's a good practice Selahaddin. But our database is connected. So we don't need this hard coded data.